profile.md

```diff
@@ -30,39 +68,39 @@ As a single organisation, we aim to strengthen collaborations
 between the different sub-organisations, and at the same time increase the
 awareness among our users on the capabilities of our "sister" projects.
 
-## Tags
-python, science, solar physics, astronomy, visualisation, atomic physics
+## Proposals
+*Guidance for students on how to apply to your organization. Should include any prerequisites or requirements. You may wish to include a template or tips for their proposals. May include limited Markdown.*
```

Comment:
we need to get here a list of stuff (we can reuse what's was asked by PSF last year and add something we think it's important)
- PR done to apply
- schedule/plan
- ...